otx/api/entities/image.py

```diff
@@ -76,13 +79,15 @@ def numpy(self) -> np.ndarray:
         """
         if self.__data is None:
             return cv2.cvtColor(cv2.imread(self.__file_path), cv2.COLOR_BGR2RGB)
+        if callable(self.__data):
+            return self.__data()
         return self.__data
 
     @numpy.setter
     def numpy(self, value: np.ndarray):
         self.__data = value
         self.__file_path = None
-        self.__height, self.__width = self.__get_size()
+        self.__size = self.__get_size()
```

Comment:
Could you check the effect from Geti? this attribute maybe used in Geti side also.

Reply:
Reverted in https://github.com/openvinotoolkit/training_extensions/pull/2009/commits/f993b6607011b4d6740ccf971e3408a8f3a8de6d